4.component.md

```diff
@@ -1,54 +1,28 @@
 
-## Component Schematics
+## Component 
 
-The role of a component schematic is to permit developers to declare, in infrastructure-neutral format, the operational characteristics of a discrete unit of execution.
+The role of a component is to permit developers to declare, in infrastructure-neutral format, the operational characteristics of a discrete unit of execution.
 
 For example, a single microservice may be modeled as a component.
 
-## Units for Time, CPU, Memory, and Disk
-
-In places in this specification, certain units of measure are applied. This section describes those units of measure.
-
-### Timing (Intervals)
-
-For timing, the default unit of time is seconds, represented as an integer.
-
-### CPU count
-
-CPU count is represented as a floating point number, where `1` means one CPU, `2` means 2 CPUs, and `0.5` means half of a CPU.
-
-The exact meaning of this unit varies from platform to platform.  Implementors should consider that logical cpu is equivalent to one AWS vCPU or one Azure vCore or one GCP Core or one IBM vCPU. Fractional values are allowed. If a runtime does not support fractional units, it MUST round up (ceiling function) to the next integer value.
-
-### Memory and Disk
-
-Memory and disk space use the notation for bytes/kilo/mega/giga/tera/peta by just using the major unit:
-
-- `1024` is 1024 bytes
-- `88K` is 88 kilobytes
-- `5M` is 5 megabytes
-- `7G` is 7 gigabytes
-- `100T` is 100 terabytes
-- `9999P` is 9999 petabytes
-
-If a `B` is appended after the unit letter, it MUST be ignored. Thus, `5M` and `5MB` are treated as identical. Case is unimportant. `15k` and `15K` MUST be treated as the same value.
-
 ## Representation
 
 All component schematics are canonically represented as JSON and can be validated using the provided JSON Schema. Because it is more amenable for human consumption, YAML is also supported, and most of the examples are given in YAML. For the sake of validation, YAML may be converted to an equivalent JSON representation before validating it against the JSON Schema.
 
 ### Top-Level Attributes
 
-The following attributes are common across all schemata defined in this document. This structure is designed to allow object type to be determined by scanning two fields (`apiVersion` and `kind`), and provide access to common fields in the `metadata` section. The `spec` section is where type-specific information is located, and in this section the `spec` section defines a `Component`.
+These attributes provide top-level information about the component definition.
```

Comment:
I'm not quite understand why this attributes is about **component definition** not **component**